src/store/sdk.ts

```diff
@@ -13,9 +15,17 @@ export default defineStore('sdk', {
           ? 'http://citadel-dev.local'
           : window.location.origin,
       ),
-      //citadel: new Citadel('https://node.runcitadel.space'),
+      userStore: useUserStore(),
```

Comment:
Have you tested this? I'm not sure if it would work because this depends on the userStore now, wich in return depends on this store, so not sure if this recursion causes an issue.

Reply:
I don't really know what Pinia does under the hood, I don't think it's recursion, just bit of spaghetti code. It works though.